docs/src/guide/installation.md

```diff
@@ -3,174 +3,310 @@
 with modm:
 
-- [Python 3](http://www.python.org/)
-- [Software Construct][scons] or [CMake][]
-- [Library Builder][lbuild]
-- AVR toolchain: [avr-gcc][] and [avrdude][]
-- ARM toolchain: [gcc-arm-toolchain][] and [OpenOCD][] (HEAD version!).
-- Optional: [Doxypress][] (preferred) or [Doxygen][]
-- Optional: [gdbgui][] for IDE-independent debugging
+- [Python 3](http://www.python.org/).
+- [Software Construct][scons] or [CMake][].
+- [Library Builder][lbuild].
+- AVR toolchain: [avr-gcc][] and [avrdude][].
+- ARM toolchain: [gcc-arm-toolchain][] and [OpenOCD][] (at least v0.11!).
+- Optional: [Doxypress][].
+- Optional: [gdbgui][] for IDE-independent debugging.
 
 Note that the modm examples use the SCons build system by default, however,
-you are not *required* to use it. See [the reference manual](../../reference/build-systems) for
-additional build system documentation.
+you are not *required* to use it. See [the reference manual](../../reference/build-systems)
+for additional build system documentation.
 
 !!! info "Use GCC 10 or newer"
-	modm uses C++20, so you need *at least* GCC 10.
+    modm uses C++20, so you need *at least* GCC 10.
 
 !!! warning "Beware of AVRs"
-	We **strongly discourage** using AVRs for new designs, due to a significant
-	lack of commitment from Atmel on keeping their AVR toolchain up-to-date.
+    We **strongly discourage** using AVRs for new designs, due to a significant
+    lack of commitment from Atmel on keeping their AVR toolchain up-to-date.
 
 !!! tip "Check your Locale"
-	If you get a `UnicodeEncodeError` when calling `lbuild`, you may need to add
-	`export LC_ALL=C.UTF-8` to your `.bashrc`. Consult the internet until
-	`python3 -c "import sys; print(sys.stdout.encoding)"` returns `UTF-8`.
+    If you get a `UnicodeEncodeError` when calling `lbuild`, you may need to add
+    `export LC_ALL=C.UTF-8` to your `.bashrc`. Consult the internet until
+    `python3 -c "import sys; print(sys.stdout.encoding)"` returns `UTF-8`.
 
 Please help us [keep these instructions up-to-date][contribute]!
 
 
 ## Linux
 
-For Ubuntu 20.04LTS, these commands install the basic build system:
+For Ubuntu 20.04LTS, these commands install the minimal build system:
 
-	sudo apt install python3 python3-pip scons cmake git doxygen
-	pip3 install modm gdbgui
+```sh
+sudo apt install python3 python3-pip scons git
+pip3 install modm
+```
+
+We recommend using a graphical frontend for GDB called [gdbgui][]:
+
+```sh
+pip3 install gdbgui
+```
 
 !!! info "Python Packages in PATH"
-	The pip3 command installs executables into `~/.local/bin`, which
-	must be added to PATH if it is not already the case.
-	Add the following line to the end of your `~/.bashrc` file:
+    The pip3 command installs executables into `~/.local/bin`, which
+    must be added to PATH if it is not already the case.
+    Add the following line to the end of your `~/.bashrc` file:
+
+    ```sh
+    export PATH="~/.local/bin:$PATH"
+    ```
```

Comment:
Actually the default `~/.profile` file on Ubuntu 20.04 includes this snippet:
```bash
# set PATH so it includes user's private bin if it exists
if [ -d "$HOME/.local/bin" ] ; then
    PATH="$HOME/.local/bin:$PATH"
fi
```

On Ubuntu it is usually sufficient to restart the bash session or manually execute `source ~/.profile`, because the `pip` command above created `~/.local/bin/`.

(I would still keep the `export PATH=...` hint.)